lzy/allocator/src/main/java/ai/lzy/allocator/services/AllocatorService.java

```diff
@@ -83,11 +102,20 @@ public AllocatorService(VmDao vmDao, @Named("AllocatorOperationDao") OperationDa
         this.allocationContext = allocationContext;
         this.config = config;
         this.cacheConfig = cacheConfig;
+        this.mountConfig = mountConfig;
 
         restoreRunningActions();
     }
 
     private void restoreRunningActions() {
```

Comment:
you may wish to introduce a new component "RecoverOperations" and move this logic there

Reply:
Moved to `RestoreOperations`